user_management/api/templates/user_management/account_validation_email.html

```diff
@@ -1,12 +1,13 @@
-{% spaceless %}{% load i18n %}{% autoescape off %}{% blocktrans with name=site.name %}
-You are receiving this email because your email address has been used to
-register an account at {{ name }}.
+{% load i18n %}
+{% blocktrans with name=site.name %}
+<p>You are receiving this email because your email address has been used to
+register an account at {{ name }}.</p>
```

Comment:
I'm wondering if we should split the template the other way around between the `p` tag and the blocktrans, eg:
```html
<p>
    {% blocktrans with name=site.name %}You are receiving this email...{% endblocktrans %}
</p>

<p>
    {% trans 'Please click the following link to complete your registration:' %}
</p>
```